sdks/java/io/tika/build.gradle

```diff
@@ -22,8 +22,8 @@ applyJavaNature(automaticModuleName: 'org.apache.beam.sdk.io.tika')
 description = "Apache Beam :: SDKs :: Java :: IO :: Tika"
 ext.summary = "Tika Input to parse files."
 
-def tika_version = "1.24.1"
-def bndlib_version = "1.43.0"
```

Comment:
Was it intended to include this?

Could reference BEAM-5542 in the commit message

Reply:
Well, I updated this one because `tika-core` has it as [provided dependencies](https://mvnrepository.com/artifact/org.apache.tika/tika-core/1.26) 

So yes, I'll update commit message, thanks.